hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java

```diff
@@ -947,45 +952,12 @@ private TcpClientConnection onAuthenticated(TcpClientConnection connection,
             connection.setRemoteUuid(response.memberUuid);
             connection.setClusterUuid(response.clusterId);
 
-            if (tpcPorts == null || tpcPorts.isEmpty()) {
+            if (!isAltoAwareClient || tpcPorts == null || tpcPorts.isEmpty()) {
                 logger.info("TPC Client: disabled, no TPC ports detected");
             } else {
+                logger.info("TPC Client: connecting to ports (" + tpcPorts.size() + "): " + tpcPorts);
```

Comment:
```suggestion
                logger.info("Alto aware Client: connecting to ports (" + tpcPorts.size() + "): " + tpcPorts);
```
I would recommend that we be consistent to use alto or TPC.

Reply:
Another comment: I think that you should also print the connection info so that we can associate the port numbers for the member.

Reply:
I have already changed lots of places from TPC to Alto in my second PR. Would you be OK if I do these changes in there as well?

Reply:
let's put connection info.